### DISCUSSION TITLE
Mobile error/note feature

### DISCUSSION THREAD

**Discussion by benastahl** [community/community#4751] [Mobile] [answered]
The ability to add a note or error in the code you are viewing if you do not currently have access to a computer to edit the code yourself.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion @thevolt6 – we do hope to get around to native code editing in the future, although it's not on the roadmap yet. I'll add your +1 to our internal tracking issue in the meantime 😊
